lib/inspec/rspec_json_formatter.rb

```diff
@@ -238,27 +238,60 @@ def format_example(example)
 class InspecRspecCli < InspecRspecJson # rubocop:disable Metrics/ClassLength
   RSpec::Core::Formatters.register self, :close
 
-  COLORS = {
-    'critical' => "\033[38;5;9m",
-    'major'    => "\033[38;5;208m",
-    'minor'    => "\033[0;36m",
-    'failed'   => "\033[38;5;9m",
-    'passed'   => "\033[38;5;41m",
-    'skipped'  => "\033[38;5;247m",
-    'reset'    => "\033[0m",
-  }.freeze
-
-  INDICATORS = {
-    'critical' => '  ×  ',
-    'major'    => '  ∅  ',
-    'minor'    => '  ⊚  ',
-    'failed'   => '  ×  ',
-    'skipped'  => '  ↺  ',
-    'passed'   => '  ✔  ',
-    'unknown'  => '  ?  ',
-    'empty'    => '     ',
-    'small'    => '   ',
-  }.freeze
+  case RUBY_PLATFORM
+  when /windows|mswin|msys|mingw|cygwin/
```

Comment:
Doing a bit more digging, it seems that for the UTF8 symbols to display, you need to switch to UTF8 with ```chcp 65001``` and then change the terminal font to one that includes emojis (such as Segoe UI Symbol included in Windows 10 Anniversary edition).

I also did a bit of digging around autodetection of terminal type or color support. ANSI does define some queries but most of the Windows terminals ignore them (despite the sequences being documented on MSDN)